corehq/apps/repeaters/models.py

```diff
@@ -163,12 +163,16 @@ def get_payload_generator(self, payload_format):
         gen = RegisterGenerator.generator_class_by_repeater_format(self.__class__, payload_format)
         return gen(self)
 
+    @property
+    @memoized
```

Comment:
I said it does "exactly the same thing". The only really unlikely thing I can think of that's different is that with this memoized property, every time we want to use _a_ payload generator, we'll now be using _the same_ one (as in the same instance). I can't imagine that being a problem; if it is, we can easily remove the `@memoized` decorator to fix that.